### ISSUE TITLE
Accessibility. Flag icon image missing alt=''

### ISSUE THREAD

**Issue by pdkn** [goveo/react-international-phone#86]
**Describe the bug**
The flag icon is missing an empty alt tag. should be a super quick fix ;)

**To Reproduce**

1. Go to 'https://react-international-phone-docs.vercel.app/'
2. Click on 'Dev tools' & run Google Lighthouse report on Accessibility
3. See error "Image elements do not have [alt] attributes"

**Expected behavior**
There should be no Accessibility issues

**Screenshots**
<img width="752" alt="Screenshot 2023-08-22 at 10 44 25" src="https://github.com/goveo/react-international-phone/assets/186148/177d610a-af0c-4bdb-ac43-4eabd47407f6">

**Comment by goveo** [comment]
@pdkn Maybe alt should contain country iso instead of empty value?
Like `alt="us-flag"` for US
What do you think?

**Comment by pdkn** [comment]
maybe, I would assume the parent component would control the focus and aria
labels, so, in this case, the actual graphic alt is not needed.
